week-02-reflections/crai-stat6863.md

```diff
@@ -3,10 +3,61 @@
 
 
 
-[place your questions/answers here, and delete this line]
+[Course Structure Quiz, Problem 9.1]
```

Comment:
Since these problems come from the textbook, please replace "Course Structure Quiz" throughout with the corresponding chapter number, or with "Course Textbook"